test/integration/smart_answer_flows/uk_benefits_abroad_test.rb

```diff
@@ -379,6 +379,103 @@ class UKBenefitsAbroadTest < ActiveSupport::TestCase
           end
         end
       end
+
+      context "answer Ireland" do
+        setup do
+          add_response :ireland
+        end
+        should "ask you if you are British or Irish?" do
+          assert_current_node :is_british_or_irish?
+        end
+        context "answer yes" do # Are you a British or Irish citizen
+          setup do
+            add_response :yes
+          end
+          should "go to WFP Ireland outcome" do
+            assert_current_node :wfp_ireland_outcome
+          end
+        end
+        context "answer no" do # Are you a British or Irish citizen
+          setup do
+            add_response :no
+          end
+          should "Ask have you ever worked in an EU country, Norway, Iceland, Liechtenstein or Switzerland?" do
+            assert_current_node :worked_in_eea_or_switzerland?
+          end
+          context "answer Yes before Jan 2021" do # Worked in EU?
+            setup do
+              add_response :before_jan_2021
+            end
+            should "go to outcome WFP EEA maybe" do
+              assert_current_node :wfp_going_abroad_eea_maybe_outcome
+            end
+          end
+          context "answer Yes after January 2021" do # Worked in EU?
+            setup do
+              add_response :after_jan_2021
+            end
+            should "ask has one of your parents ever lived in an EU country, Norway, Iceland, Liechtenstein or Switzerland?" do
+              assert_current_node :parents_lived_in_eea_or_switzerland?
+            end
+            context "answer Yes before January 2021" do # parents EU?
+              setup do
+                add_response :before_jan_2021
+              end
+              should "go to outcome WFP EEA maybe" do
+                assert_current_node :wfp_going_abroad_eea_maybe_outcome
+              end
```

Comment:
General point throughout this PR - you could make the code easier to scan by changing it to:

```ruby
should "go to outcome WFP EEA maybe" do
  add_response :before_jan_2021
  assert_current_node :wfp_going_abroad_eea_maybe_outcome
end
```

`setup` blocks are only really useful if you then have multiple `should` blocks. But if you only have one `should` block, you may as well put the code inline.

Reply:
I think this point still stands - by far the biggest change in this PR is this test file, (2082 additions), which could probably be reduced to maybe 1000 additions by squishing some of the unnecessary `setup` blocks. The next person to edit this Smart Answer would thank you!